app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,91 @@
+<%# apps/views/recipe/show.html.erb %>
+<%= render 'layouts/header' %>
+
+<h1 class="recipe_name"><%= @recipe.name %></h1>
+<p><strong>Preparation Time:</strong> <%= @recipe.prescription_time %> Hours</p>
```

Comment:
- [ ] Good job fixing the preparation time format as requested by the previous reviewer [here](https://github.com/tomdan-ai/Recipe-App/pull/19#discussion_r1291987730), however, the `Cooking time` is missing now kindly fix this

- [ ] Also the description of the recipe is not being added, kindly fix this so it shows upon creating the recipe.
Your design
![image](https://github.com/tomdan-ai/Recipe-App/assets/81932478/9e9dc23a-35c4-407c-b815-aa6e9a7983ca)

Given Wireframe
![image](https://github.com/tomdan-ai/Recipe-App/assets/81932478/f46c52d3-83c1-4ea3-9bd3-b6392b51ffd5)


- [ ] (optional) This requirement was missed by the previous reviewer, but kindly consider adding the part that is supposed to generate a shopping list for the recipe using the food list for a given user.

screenshot of requirements
![image](https://github.com/tomdan-ai/Recipe-App/assets/81932478/c2e284c9-fefa-4ecf-bb5e-9a5f8c2f9447)